strings/manacher.py

```diff
@@ -21,6 +24,8 @@ def palindromic_string(input_string: str) -> str:
     new_input_string = ""
     output_string = ""
 
+    i: Union[int, str]

```

Comment:
I think `i` is a loop variable and that's the reason `mypy` must have been giving some error. Instead, you should change the loop variable for other loops to `j`, `k`, etc.

Reply:
I have changed all the loop variables accordingly